tests/Aql/EngineInfoContainerCoordinatorTest.cpp

```diff
@@ -272,7 +273,8 @@ TEST(EngineInfoContainerTest,
 
   // Mock the Registry
   fakeit::When(Method(mockRegistry, insert))
-      .Do([&](QueryId id, Query* query, double timeout, bool isPrepared, bool keepLease) {
+    .Do([&](QueryId id, Query* query, double timeout, bool isPrepared, bool keepLease,
+            std::unique_ptr<arangodb::cluster::CallbackGuard>&) {
```

Comment:
I do not get how this works, the guard is not used so it seem to be ok. But shouldn't it go in as rvalue reference?